### PR TITLE
Add ContentaCMS.

### PR DESCRIPTION
**Pull request by mherchel**
Adding ContentaCMS.

### CODE REVIEW

File: site/content/projects/contenta.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+typeofcms: "API Driven"
+supportedgenerators:
+  - All
+description: Contenta is the community driven API distribution for Drupal 8.
+---
+## Contenta is a decoupled Drupal 8 distribution
+
+Contenta is the community driven API distribution for Drupal 8, and provides the robustness and flexibility of Drupal in a friendly API-first package.
+
+It provides a standard platform that is API ready along with demo content and example front-end applications. Contenta intends to ease the pain of using, or simply trying, decoupled Drupal.
```
Comment (dawehner) [inline site/content/projects/contenta.md]: It would be nice to also mention the documentation aspect

Comment (mherchel) [reply site/content/projects/contenta.md]: 👍 Done!

Comment (dawehner) [reply site/content/projects/contenta.md]: 👍

File: site/content/projects/contenta.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Free Open Source Software to be hosted anywhere
+
+Contenta is based of of Drupal, which is a powerful, robust content management system built on object-oriented PHP. Drupal runs websites such as whitehouse.gov, grammy.com, harvard.edu, and millions more.
```
Comment (verythorough) [inline site/content/projects/contenta.md]: typo: "based of of Drupal"

Comment (mherchel) [reply site/content/projects/contenta.md]: Fixed :) Thanks!

**Comment by verythorough** [comment]
Awesome. Thank you!

**Pull request merged**